src/HtmlGenerator/Utilities/Markup.cs

```diff
@@ -236,6 +236,7 @@ public static void WriteLinkPanel(Action<string> writeLine, params string[] rows
                 writeLine(row);
             }
 
+            writeLine(string.Format("<tr><td>File indexed on: {0}</td></tr>", DateTime.Now.ToString()));

```

Comment:
We must have a CSS style for gray somewhere, if not we should mark this string as gray via CSS.